components/landingPage/faq.js

```diff
@@ -3,6 +3,13 @@ import styled from "styled-components";
 import { Accordion } from "@moneypensionservice/directories";
 
 // styled components
+
+const AEMAccordion = styled(Accordion)`
+    border-bottom: 1px solid #106098;
+     &:nth-child(1) {
+        border-bottom:1px solid #106098 !important;
+    }
```

Comment:
I'm not sure the `nth-child` rule is adding anything here. 
According to the design we could do with a border on the first element though. 
Also the colour should be updated to magenta-like hue.

Reply:
Fixed. Thanks for paying attention. Dont know how I missed that. However, I think the design got the direction of the accordion wrong. It seems logical for the accordion to face right when they are not opened and down when they are.

Reply:
OK, let's see if anything else gets picked up in the QA process. I will approve as it is now though.